android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -84,7 +85,15 @@ class Screen constructor(context: ReactContext?) : FabricEnabledViewGroup(contex
     }
 
     val headerConfig: ScreenStackHeaderConfig?
-        get() = getChildAt(0) as? ScreenStackHeaderConfig
+        get() {
+            for (child in children) {
+                if (child is ScreenStackHeaderConfig) {
+                    return child
+                }
+            }
+
+            return null
+        }
```

Comment:
we can have it done in more sassy way
```suggestion
        get() = children.find { it is ScreenStackHeaderConfig } as? ScreenStackHeaderConfig
```

Reply:
Thanks for the hint! Resolved in https://github.com/software-mansion/react-native-screens/pull/1883/commits/74c4f962631dfc1b0b0e6c95a8cf025b6032245b.